src/Driver/OCI8/Statement.php

```diff
@@ -72,10 +72,14 @@ public function bindParam($param, &$variable, $type = ParameterType::STRING, $le
         }
 
         if ($type === ParameterType::LARGE_OBJECT) {
-            $lob = oci_new_descriptor($this->connection, OCI_D_LOB);
-            $lob->writeTemporary($variable, OCI_TEMP_BLOB);
+            if ($variable !== null) {
+                $lob = oci_new_descriptor($this->connection, OCI_D_LOB);
+                $lob->writeTemporary($variable, OCI_TEMP_BLOB);
 
-            $variable =& $lob;
+                $variable =& $lob;
+            } else {
+                $type = ParameterType::STRING;
```

Comment:
Just out of curiosity, why did you choose `ParameterType::STRING` here over `ParameterType::NULL`?

Reply:
There’s no special handling of the null parameters in oci8.